core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -54,69 +55,47 @@ var LearnerDashboardPage = function() {
   };
 
   this.navigateToCompletedSection = async function() {
-    await waitFor.elementToBeClickable(
-      completedSection, 'Completed tab takes too long to be clickable');
-    await completedSection.click();
+    await action.click('Completed tab', completedSection);
   };
 
   this.navigateToInCompleteSection = async function() {
-    await waitFor.elementToBeClickable(
-      incompleteSection, 'In Progress tab takes too long to be clickable');
-    await incompleteSection.click();
+    await action.click('In Progress tab', incompleteSection);
   };
 
   this.navigateToIncompleteCollectionsSection = async function() {
-    await waitFor.elementToBeClickable(
-      incompleteCollectionsSection,
-      'Incomplete Collection Section tab takes too long to be clickable');
-    await incompleteCollectionsSection.click();
+    await action.click(
+      'In Progress Collections tab', incompleteCollectionsSection);
   };
 
   this.navigateToIncompleteExplorationsSection = async function() {
-    await waitFor.elementToBeClickable(
-      incompleteExplorationsSection,
-      'Incomplete Collection Section tab takes too long to be clickable');
-    await incompleteExplorationsSection.click();
+    await action.click(
+      'In Progress Explorations tab', incompleteExplorationsSection);
   };
 
   this.navigateToCompletedCollectionsSection = async function() {
-    await waitFor.elementToBeClickable(
-      completedCollectionsSection,
-      'Completed Collection Section tab takes too long to be clickable');
-    await completedCollectionsSection.click();
+    await action.click(
+      'Completed Collections tab', completedCollectionsSection);
   };
 
   this.navigateToCompletedExplorationsSection = async function() {
-    await waitFor.elementToBeClickable(
-      completedExplorationsSection,
-      'Completed Collection Section tab takes too long to be clickable');
-    await completedExplorationsSection.click();
+    await action.click(
+      'Completed Explorations tab', completedExplorationsSection);
   };
 
   this.navigateToFeedbackSection = async function() {
-    await waitFor.elementToBeClickable(
-      feedbackSection, 'Feedback Section tab takes too long to be clickable');
-    await feedbackSection.click();
+    await action.click('Feedback tab', feedbackSection);
   };
 
   this.navigateToFeedbackThread = async function() {
-    await waitFor.elementToBeClickable(
-      feedbackThread, 'Feedback Thread tab takes too long to be clickable');
-    await feedbackThread.click();
+    await action.click('Feedback Thread tab', feedbackThread);
   };
 
   this.navigateToSubscriptionsSection = async function() {
-    await waitFor.elementToBeClickable(
-      subscriptionsSection,
-      'Subscriptions Section tab takes too long to be clickable');
-    await subscriptionsSection.click();
+    await action.click('Subscriptions tab', subscriptionsSection);
   };
 
```

Comment:
@StephanieGreenberg I know this PR is just replacing `click()` with `action.click()`, but there's actually only one other change we need to make to mark this file as complete in #10798: Add a `waitFor.visibilityOf()` call in `expectTitleOfExplorationSummaryTileToMatch`. Here's all that needs to be done:

```diff
   this.expectTitleOfExplorationSummaryTileToMatch = async function(title) {
     // This explorationTitleArray is an Array but it will have only one element
     // that is the exploration with the title passed as a parameter.
     var explorationTitle = element(
       by.cssContainingText('.protractor-test-exp-summary-tile-title', title));
+    waitFor.visibilityOf(explorationTitle, 'Exploration title takes too long to appear');
     expect(await explorationTitle.getText()).toMatch(title);
   };
```

Would you mind making that change too? Then you'll have finished the whole file!

Reply:
@U8NWXD PTAL